src/Kunstmaan/AdminBundle/Helper/Security/Acl/Permission/PermissionAdmin.php

```diff
@@ -232,7 +232,7 @@ public function bindRequest(Request $request)
             $user = $this->tokenStorage->getToken()->getUser();
             $this->createAclChangeSet($this->resource, $changes, $user);
 
-            $cmd = 'php ' . $this->kernel->getRootDir() . '/../bin/console kuma:acl:apply';
+            $cmd = 'php ' . $this->kernel->getProjectDir() . '/../bin/console kuma:acl:apply';
```

Comment:
`../` should be removed?

Reply:
You're right! I've fixed the code and extended the unit test to actually cover this part